From 7839b53be3677ed1d0ac92c8c59badfc05287a62 Mon Sep 17 00:00:00 2001
From: Moritz Marquardt <moritz.marquardt@zeiss.com>
Date: Thu, 5 Dec 2019 14:26:57 +0000
Subject: [PATCH] Configure u-boot env. in sd-card for jetson-nano

---
 include/configs/p3450-porg.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/p3450-porg.h b/include/configs/p3450-porg.h
index a113bcbc58..f1e0965101 100644
--- a/include/configs/p3450-porg.h
+++ b/include/configs/p3450-porg.h
@@ -31,8 +31,8 @@
 #define BOOT_TARGET_DEVICES(func) \
 	func(MMC, mmc, 1)
 
-/* Environment s/b at end of SPI, fix it later */
-#define CONFIG_ENV_IS_NOWHERE
+/* Env is located in a partition of the sd-card (mmc device in u-boot) */
+#define CONFIG_ENV_IS_IN_MMC
 
 /* SPI */
 #define CONFIG_SF_DEFAULT_MODE		SPI_MODE_0
